tracer.go

```diff
@@ -266,3 +264,30 @@ func (t Tags) Apply(o *StartSpanOptions) {
 		o.Tags[k] = v
 	}
 }
+
+// Tag may be passed as a StartSpanOption to add a tag to new spans,
+// or its Set method may be used to apply the tag to an existing Span,
+// for example:
+//
+// tracer.StartSpan("opName", Tag{"Key", value})
```

Comment:
This renders wrong in godoc. The example code should be indented, the "or" should not.